package.json

```diff
@@ -8,7 +8,7 @@
     "@radix-ui/react-accordion": "^1.2.1",
     "@radix-ui/react-checkbox": "^1.1.2",
     "@radix-ui/react-label": "^2.1.0",
-    "@radix-ui/react-radio-group": "^1.2.1",
```

Comment:
Since the "@radix-ui/react-radio-group" package is uninstalled, src/components/ui/radio-group.tsx needs to be deleted since it imports this package. Or the package can be reinstalled